lib/ohai/plugins/rust.rb

```diff
@@ -0,0 +1,32 @@
+# Author:: Christopher M Luciano (<cmlucian@us.ibm.com>)
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Ohai.plugin(:Rust) do
+  provides "languages/rust"
+
+  depends "languages"
+
+  collect_data do
+    output = nil
+
+    rust = Mash.new
+    so = shell_out("rustc --version")
+    if so.exitstatus == 0
```

Comment:
@lamont-granquist I tried it without the conditional and then noticed that my "should not set rust if rust command fails" test failed. When I tried to run "rustc --version" on a computer w/o rust installed I didn't get my expected output but instead "command not found: rustc" with an exit code of 127. So I still get valid text from stdout assigned to my output variable.

I think that it would still require validation of sorts with a conditional if I didn't check the exit code.